packages/nextcloud/lib/src/api/spreed.openapi.json

```diff
@@ -0,0 +1,16719 @@
+                    "avatarVersion": {
+                        "type": "string"
+                    },
+                    "breakoutRoomMode": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "breakoutRoomStatus": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "callFlag": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "callPermissions": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "callRecording": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "callStartTime": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "canDeleteConversation": {
+                        "type": "boolean"
+                    },
+                    "canEnableSIP": {
+                        "type": "boolean"
+                    },
+                    "canLeaveConversation": {
+                        "type": "boolean"
+                    },
+                    "canStartCall": {
+                        "type": "boolean"
+                    },
+                    "defaultPermissions": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "description": {
+                        "type": "string"
+                    },
+                    "displayName": {
+                        "type": "string"
+                    },
+                    "hasCall": {
+                        "type": "boolean"
+                    },
+                    "hasPassword": {
+                        "type": "boolean"
+                    },
+                    "id": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "isCustomAvatar": {
+                        "type": "boolean"
+                    },
+                    "isFavorite": {
+                        "type": "boolean"
+                    },
+                    "lastActivity": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "lastCommonReadMessage": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "lastMessage": {
+                        "oneOf": [
+                            {
+                                "$ref": "#/components/schemas/ChatMessage"
+                            },
+                            {}
+                        ]
+                    },
```

Comment:
The spec is broken as a ChatMessage also matches an empty object

Reply:
The empty array problem strikes again...